src/components/ScmRepositoryDetailPage/ScmRepositoryDetailPage.tsx

```diff
@@ -0,0 +1,145 @@
+  const serviceContainerScmRepositoryRunner = serviceContainerScmRepository.run;
+
+  useEffect(() => {
+    serviceContainerScmRepositoryRunner({ serviceData: { id: scmRepositoryId } });
+  }, [serviceContainerScmRepositoryRunner, scmRepositoryId]);
+
+  useTitle(
+    serviceContainerScmRepository.data?.id
+      ? `${serviceContainerScmRepository.data.id} | ${PageTitles.repositories}`
+      : `Error loading ${PageTitles.projectDetail}`
+  );
+
+  const internalScmRepositoryLink = (internalUrl: string) => {
+    /**
+     * Parses internal repo url to Gerrit gitweb link of the project
+     */
+    const parseInternalRepoLink = (url: string) => {
+      let protocol = url.split('://')[0];
+      let base = url.split('://')[1].split('/')[0];
+      let project = url.split(base + (['https', 'http'].includes(protocol) ? '/gerrit/' : '/'))[1];
+      return 'https://' + base + '/gerrit/gitweb?p=' + project + ';a=summary';
+    };
+    return (
+      <div>
+        <Split hasGutter>
+          <SplitItem isFilled>
+            <ClipboardCopy isReadOnly hoverTip="Copy" clickTip="Copied" maxWidth="200px">
+              {internalUrl}
+            </ClipboardCopy>
+          </SplitItem>
+          <SplitItem>
+            <Button
+              component="a"
+              href={parseInternalRepoLink(internalUrl)}
+              target="_blank"
+              rel="noopener noreferrer"
+              variant="tertiary"
+              icon={<ExternalLinkAltIcon />}
+            >
+              Gerrit
+            </Button>
+          </SplitItem>
+        </Split>
+      </div>
+    );
+  };
+
+  const parseScmRepositoryTitle = (internalUrl: string) => (internalUrl ? internalUrl.split('/').splice(3).join('/') : '');
+
+  const attributes = [
+    {
+      name: (
+        <span>
+          Internal SCM URL{' '}
+          <Tooltip removeFindDomNode content={<div>URL of the internal SCM from Gerrit</div>}>
+            <InfoCircleIcon />
+          </Tooltip>
+        </span>
+      ),
+      value: serviceContainerScmRepository.data?.internalUrl
+        ? internalScmRepositoryLink(serviceContainerScmRepository.data?.internalUrl)
+        : undefined,
+    },
+    {
+      name: (
+        <span>
+          External SCM URL{' '}
+          <Tooltip removeFindDomNode content="URL of the external SCM">
+            <InfoCircleIcon />
+          </Tooltip>
+        </span>
+      ),
+      value: serviceContainerScmRepository.data?.externalUrl ? (
+        <ClipboardCopy isReadOnly hoverTip="Copy" clickTip="Copied">
+          {serviceContainerScmRepository.data?.externalUrl}
+        </ClipboardCopy>
+      ) : undefined,
+    },
+    {
+      name: (
+        <span>
+          Pre-build Sync{' '}
+          <Tooltip
+            removeFindDomNode
+            content={<div>Whether the internal repository will be automatically updated just before each build</div>}
```

Comment:
Use `Option declaring whether the Pre-build Synchronization (for example adding new commits) from the external repository to the internal repository should happen or not.`